docs/Example HOWTO.md

```diff
@@ -232,68 +223,140 @@ dns2.example.com. 3600    IN      A       192.168.0.20
 ;; WHEN: Tue Jul 13 19:58:50 IST 2021
 ;; MSG SIZE  rcvd: 134
 ```
+### Using Dig to Browse NMOS Related DNS Items
 
-### Checking the `SRV` records
+We can use `dig` to do a top down browse of the NMOS services being served by BIND9. First we simply 
+check if our DNS is offering up the NMOS services by looking at all the pointers to services our DNS is 
+providing:
+
+```
+$dig _services._dns-sd._udp.example.com PTR
+; <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.amzn2.5.2 <<>> PTR _services._dns-sd._udp.example.com
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 4132
+;; flags: qr aa rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 1, ADDITIONAL: 2
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;_services._dns-sd._udp.example.com. IN	PTR
+
+;; ANSWER SECTION:
+_services._dns-sd._udp.example.com. 3600 IN PTR	_nmos-query._tcp.example.com.
+_services._dns-sd._udp.example.com. 3600 IN PTR	_nmos-register._tcp.example.com.
+
+;; AUTHORITY SECTION:
+example.com.		3600	IN	NS	dns1.example.com.
+
+;; ADDITIONAL SECTION:
+dns1.example.com.	3600	IN	A	192.168.0.18
+
+;; Query time: 0 msec
+;; SERVER: 10.0.50.59#53(10.0.50.59)
+;; WHEN: Fri Sep 02 13:02:59 UTC 2022
+;; MSG SIZE  rcvd: 158
+
+```
+
+We see in the ANSWER section that our DNS has the two entries we included in our BIND9 database file for our NMOS registry and NMOS query type services.
+
+
+### Checking the `PTR` records
 
-We can also use `dig` to check the presence of the `_nmos._register_.tcp` record:
 
+Next we use dig to query the DNS for PTR records associated 
+with the entries obtained in our last step. 
 
 ```
-gp@example.com:~ # dig @localhost _nmos-register._tcp.example.com SRV
+$dig _nmos-register._tcp.example.com PTR
```

Comment:
And specifically this example which was relying on those invalid SRV records to skip the PTR indirect. This looks better to me, thanks.

Reply:
Yep